apps/web/pages/privacy-policy.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-  <div class="p-5" v-html="getHTMLTexts()" />
+  <div class="flex items-center justify-center">
+    <div class="container p-4 overflow-x-auto" v-html="getHTMLTexts()" />
+  </div>
```

Comment:
I would do something like:
`<div class="w-full p-5 overflow-x-auto" v-html="getHTMLTexts()" />`

At least we have to change all other legal texts to the same format.
What do you think?